index.html

```diff
@@ -309,7 +309,10 @@ <h1>Important Terms</h1>
 			</dd>
 			<dt><dfn data-dfn-for="element" data-export="">Hidden</dfn></dt>
 			<dd>
-				<p>Indicates that the <a>element</a> is not visible, <a>perceivable</a>, or interactive to <em>any</em> user. An element is considered <em>hidden</em> if it or any one of its ancestor elements is not rendered or is explicitly hidden.</p>
+				<p>Indicates that the <a>element</a> is absent from the accessibility tree and is not exposed to accessibility APIs. An element is considered hidden if it or any one of its ancestor elements has been explicitly hidden, either by CSS properties <code>display:none</code>, <code>visibility:hidden</code> or <code>visibility:collapse</code>, or with the aria attribue <code>aria-hidden</code>.<p>
```

Comment:
I found it somewhat confusing to have the old content here; it feels like most of it belongs with "hidden from all users". I also think it would be good to have short definitions of terms and mostly reference the spec for details (so that we don't have to update several places).

I.e., something like:

"Indicates that the element is excluded from the accessibility tree [link to 7.1]  and therefore not exposed to accessibility APIs. For example, an element is considered hidden if it is `hidden for all users` or by using the aria attribute `aria-hidden` on it or any one of its ancestor elements ."

or even shorter:

"Indicates that the element is excluded from the accessibility tree and therefore not exposed to accessibility APIs. See also [ 7.1 ] , [ hidden for all users ] , [ aria-hidden ]"

Reply:
I'm not feeling too strongly about it since I think that there's nothing technically wrong with the text as is.

Reply:
Thanks for this feedback/perspective, I like the first slightly longer suggestion and will incorporate!